lib/manager/actions/feeds.js

```diff
@@ -141,6 +141,14 @@ export function fetchFeedSource (feedSourceId: string) {
         dispatch(fetchRelatedTables(feedSource))
         return feedSource
       })
+      .then(() => {
+        // If viewing a particular feed, navigate to a new feed version as soon as it becomes available.
+        // (If user is not looking at this feed, don't navigate away from their current page.)
+        const newVersionPath = `/feed/${feedSourceId}`
+        if (browserHistory.getCurrentLocation().pathname.startsWith(`${newVersionPath}/version/`)) {
+          browserHistory.push(newVersionPath)
+        }
+      })
```

Comment:
Oops, sorry for the confusion, I meant that this should go in the `then()` when `dispatch(fetchFeedSource())` is called in the switch statement in `actions/status.js`.

Reply:
Alright, updated in https://github.com/ibi-group/datatools-ui/pull/644/commits/e523505e7713ae582879443a8c57a0bafd70cf00.